huaweicloud/resource_huaweicloud_compute_instance.go

```diff
@@ -78,6 +78,10 @@ func ResourceComputeInstanceV2() *schema.Resource {
 				Type:     schema.TypeString,
 				Required: true,
 			},
+			"description": {
+				Type:     schema.TypeString,
+				Optional: true,
```

Comment:
`Computed` is missing

Reply:
if add Computed, we can't update the description to empty

Reply:
as discussed, Computed added